presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -1262,11 +1262,12 @@ public void testHugeIn()
         assertExecute("bound_double in (12.34E0, " + doubleValues + ")", BOOLEAN, true);
         assertExecute("bound_double in (" + doubleValues + ")", BOOLEAN, false);
 
-        String stringValues = range(2000, 7000)
-                .mapToObj(i -> format("'%s'", i))
-                .collect(joining(", "));
-        assertExecute("bound_string in ('hello', " + stringValues + ")", BOOLEAN, true);
-        assertExecute("bound_string in (" + stringValues + ")", BOOLEAN, false);
+        // RowExpression optimization is needed for following to run. Integration test already covered such scenario.
```

Comment:
accidental change?

Reply:
Nope, this test should fail if we remove the type only cast from `SqlToRowExpressionTranslator`.  But it should be fine in IntegrationTests because we will optimize the `RowExpression` before we compile it. 

However, adding `RowExpression` optimization here will do constant folding on many other tests and make tests no longer testing `ExpressionCompiler`.

Reply:
As discussed offline, let's remove the comments.